### PR TITLE
Cassandra integration with Schema Support

### PR DESCRIPTION
**Pull request by otterc**

### CODE REVIEW

**Comment by chandnisingh** [comment]
@tweise please review and merge

**Comment by chandnisingh** [comment]
@ishark Can you please review this

File: contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java [hunk 1]
```diff
@@ -45,46 +47,36 @@
...
+  protected final transient ArrayList<Object> getters;
+  protected transient Class<?> pojoClass;
 
-  /*
-   * An ArrayList of Columns in the Cassandra Table.
+  /**
+   * The input port on which tuples are received for writing.
    */
-  public ArrayList<String> getColumns()
+  @InputPortFieldAnnotation(optional = true)
```
Comment (ishark) [inline contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java]: Is this input port intended to be optional? This hides the original field in superclass.

Comment (ishark) [inline contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java]: Ignore previous comment.. Even the super class one has optional annotation :)

**Comment by chandnisingh** [comment]
This has been merged so closing it.

**Pull request closed without merging**